packages/ui/src/components/chat.tsx

```diff
@@ -0,0 +1,149 @@
+import { ChangeEvent, KeyboardEvent, ReactNode, RefObject } from 'react'
+
+import { Button, Icon, Input, Text } from '@/components'
+import { cn } from '@/utils/cn'
+
+// Root Container
+const Root: React.FC<{ children: ReactNode }> = ({ children }) => {
+  return <div className="w-full h-full overflow-y-auto flex flex-col">{children}</div>
```

Comment:
Looks like the ESLint rules are kicking up a fuss about the order of classes here.
```suggestion
  return <div className="flex size-full flex-col overflow-y-auto">{children}</div>
```

Same with other ones below. Worth running the eslint --fix command.

Reply:
<img width="1247" alt="Screenshot 2025-02-03 at 11 18 02" src="https://github.com/user-attachments/assets/a5517255-2f50-4e02-90d2-b76d28c278a5" />